aikau/src/main/resources/alfresco/css/less/defaults.less

```diff
@@ -50,6 +50,19 @@
 @de-emphasized-font-color: #999;
 @highlighted-font-color: #fff;
 
+// Dashlets
+@dashlet-background: #fff;
+@dashlet-border: 1px solid #ccc;
```

Comment:
Just a minor thing... but I was thinking that where we have a "global" variable, e.g. @standard-border we should reference it in the more specific examples, e.g 

```
@dashlet-border: @standard-border
```

That way borders can be universally changed, or you could just override borders for dashlets. Anopther example would be:

@dashlet-background = @primary-background-color

I'm not sure if there are any others, we could look to add others, like @standard-border-radius?